var/spack/repos/builtin/packages/lfortran/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Lfortran(CMakePackage):
+    """Modern interactive LLVM-based Fortran compiler"""
+
+    homepage = "https://lfortran.org"
+
+    git = "https://gitlab.com/lfortran/lfortran.git"
+
+    version("master", branch="master")
+    version("0.16.0", tag="v0.16.0")
+    version("0.15.0", tag="v0.15.0")
```

Comment:
Would using the release tarballs for these two versions mean we only have to do the fancy stuff below for GitVersion versions?

Reply:
Yes, correct. The tarballs have everything generated, so only depend on cmake, not Bison, re2c.

Reply:
Where are the tarball?

Reply:
@certik where are the tarballs?

Reply:
Ah, here is the link to all of them: https://lfortran.org/download/

Reply:
Done!